Drawsana/Data model/Drawing.swift

```diff
@@ -173,10 +191,13 @@ public class MultiDecoder<ResultType> {
 
   /// Adds the decoded result to `results` if decoding succeeds, otherwise does
   /// nothing.
-  public func tryDecoding<T: Shape>(_ type: T.Type) {
+  public func tryDecoding<T: Shape>(_ type: T.Type) throws {
```

Comment:
I would recommend to document which errors this method can throw. The comment ("...otherwise does nothing.") feels slightly confusing now.